Monika After Story/game/zz_submods.rpy

```diff
@@ -478,14 +490,35 @@ init -980 python in mas_submod_utils:
 
         return func_dict.get(_function)[0]
 
-    def setArgs(key, _function, args=[]):
+    @mas_utils.deprecated(
+        use_instead="functools.partial",
+        use_instead_msg_fmt="Wrap your callable in '{use_instead}' to provide it args/kwargs."
+    )
+    def getArgs(key, _function):
+        """
+        Gets args for the given function at the given key
+
+        IN:
+            key - key to retrieve the function from
+            _function - function to retrieve args from
+
+        OUT:
+            list of args if the function is present
+            If function is not present, None is returned
+        """
+        return __getArgs(key, _function)
+
+    @mas_utils.deprecated(
+        use_instead="functools.partial",
+        use_instead_msg_fmt="Wrap your callable in '{use_instead}' to provide it args/kwargs."
+    )
+    def setArgs(key, _function, args=None):
```

Comment:
I'm not sure how I feel about this as the intent of this function was to allow changing args after the plugin was registered, as to loosely act like a standard function. 

Given the intent of the function, I don't feel it's appropriate to deprecate it as otherwise we may as well just enforce func plugins to not have args.

Reply:
Amending this. Due to discussion, we'll keep these deprecations for the final r6 release

However in renpy 8, we will remove function plugin args **entirely**, making all of them normalized 0 arg functions, any global state should be maintained by store variables for respective submods/official uses.